R/plotFeatureLoadings.R

```diff
@@ -0,0 +1,404 @@
+#' Plot tree with feature loadings after performing a reducedDim (e.g. PCA, LDA, NMF)
```

Comment:
Remove " (e.g. PCA, LDA, NMF)" -> that cannot be on the title line. This can be moved to line 4

Reply:
This method works also without tree -> remove "tree"?